src/options/datatypes_options.toml

```diff
@@ -86,28 +86,40 @@ name   = "Datatypes Theory"
   help       = "internally use shared selectors across multiple constructors"
 
 [[option]]
-  name       = "sygusSymBreak"
+  name       = "sygusSimpleSymBreak"
   category   = "regular"
-  long       = "sygus-sym-break"
-  type       = "bool"
-  default    = "true"
-  help       = "simple sygus symmetry breaking lemmas"
-
-[[option]]
-  name       = "sygusSymBreakAgg"
-  category   = "regular"
-  long       = "sygus-sym-break-agg"
-  type       = "bool"
-  default    = "true"
-  help       = "use aggressive checks for simple sygus symmetry breaking lemmas"
-
-[[option]]
-  name       = "sygusSymBreakDynamic"
-  category   = "regular"
-  long       = "sygus-sym-break-dynamic"
-  type       = "bool"
-  default    = "true"
-  help       = "dynamic sygus symmetry breaking lemmas"
+  long       = "sygus-simple-sym-break=MODE"
+  type       = "SygusSimpleSymBreakMode"
+  default    = "AGG"
+  help       = "if and how to apply simple symmetry breaking for smart enumeration"
```

Comment:
Why is it that we call this "simple" again? It's somewhat cryptic. Maybe we could change the name at this point?

Reply:
We called it simple symmetry breaking in our CAV draft although this didnt make it to the final version.  I think it is the right name for it, it is just simple custom symmetry breaking lemmas that are added statically.   Its also the name of the module: https://github.com/cvc5/cvc5/blob/master/src/theory/datatypes/sygus_simple_sym.h#L28.

Reply:
I see. Indeed I had looked into the CAV paper and not found it, so I was confused. Maybe we could then add a simple qualifier in the helper, like "(based on grammar/conjecture structure)"?

Reply:
Its actually only based on the grammar.